iib/workers/tasks/build.py

```diff
@@ -54,11 +61,18 @@
 
 
 @retry(
-    exceptions=ExternalServiceError,
-    tries=get_worker_config().iib_total_attempts,
-    delay=get_worker_config().iib_retry_delay,
-    jitter=get_worker_config().iib_retry_jitter,
-    logger=log,
+    before_sleep=before_sleep_log(log, logging.WARNING),
+    reraise=True,
+    retry=retry_if_exception_type(ExternalServiceError),
+    stop=stop_after_attempt(get_worker_config().iib_total_attempts),
+    wait=wait_chain(
+        *[
+            wait_fixed(
+                get_worker_config().iib_retry_delay + (x * get_worker_config().iib_retry_jitter)
+            )
+            for x in range(get_worker_config().iib_total_attempts)
+        ]
+    ),
```

Comment:
Here could be used something simpler.
Currently, the waiting times are 5, 10, 15, 20, ...
If we used for example wait_exponential(multiplier=4)`, waiting times would be 4, 8, 16, 16, 32.